### PR TITLE
Brock feature

### PR DESCRIPTION
**Pull request by BrockAltug**

### CODE REVIEW

File: assets/js/playlist.js [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
-      "justify-between",
       "bg-gray-300",
       "rounded",
-      "items-center",
       "mb-2"
-    ); // add styling later
+    ); // add styling
+
+    // Create a YouTube search URL using the artist and song title
+    const youtubeSearchUrl = `https://www.youtube.com/results?search_query=${encodeURIComponent(song.trackName)}+${encodeURIComponent(song.artistName)}`;
```
Comment (rennancruz) [inline assets/js/playlist.js:24]: Nice!

File: assets/js/search.js [hunk 2]
```diff
@@ -8,80 +8,98 @@ async function searchMusic() {
       "<p class=''>Please enter a song or artist's name.</p>";
     return;
   }
+
   const itunesResponse = await fetch(
     `https://itunes.apple.com/search?term=${encodeURIComponent(
       searchInput
-    )}&entity=song&limit=100`
+    )}&entity=song&limit=25`
```
Comment (rennancruz) [inline assets/js/search.js:15]: Makes sense. Maybe we should add logic to also go to another page once we scroll through the 25 results?

File: assets/js/search.js [hunk 3]
```diff
@@ -8,80 +8,98 @@ async function searchMusic() {
...
-        </div>
-        <button id="addPlaylist" class="bg-blue-500 rounded px-3 py-1 flex text-sm text-white hover:bg-blue-600" style="flex-shrink:0;width:100px;" onclick='addToPlaylist(${JSON.stringify(
-          results
-        )})'> 
-            <i class="fas fa-plus text-gray-900 mt-3.5"></i> Add To Playlist 
-        </button>
-    
-    `;
-      resultsContainer.appendChild(songElement);
+    // Remove duplicates by using a Map to filter unique track and artist combinations
```
Comment (rennancruz) [inline assets/js/search.js:23]: Nice

**Review by rennancruz — APPROVED** [review]
LGTM!

**Review by rennancruz — APPROVED** [review]

**Review by Kwansom — APPROVED** [review]

**Pull request merged**